dev-system/support/build-for-container.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
```

Comment:
Add a comment with a description of the script and any commandline variables it accepts.